config/locales/hyrax.zh.yml

```diff
@@ -140,21 +140,21 @@ zh:
             no_delay: 没有延迟 -- 所有作品一旦存储即可发布
             title: 发布
             varies:
-              any: 允许存款人决定
-              between: 在'现在'之间
-              description: '可变的 -- 存储者可以设置一件作品的发布日期:'
+              any: 存储者可以选择任何延迟发布时间
+              between: 存储者可以选择最近时间
+              period: 存储者可以选择延迟发布期限：
```

Comment:
Can you move `period` back to its prior location, below `select`?

Reply:
Good points, @hweng! I will re-sort all of the locale files in a separate PR, since the English one has gotten out of order. Thank you. :)

Reply:
Thanks for your reviews and comments, @mjgiarlo!